spec/models/manageiq/providers/hawkular/middleware_manager/event_catcher/monitor_spec.rb

```diff
@@ -0,0 +1,42 @@
+require "manageiq/providers/hawkular/middleware_manager/event_catcher/monitor"
```

Comment:
https://gitter.im/ManageIQ/manageiq?at=570e7cfe30a23db340afb500
so, `Monitor` seems to be a bad choice :(
would you mind renaming it to `Stream`, also for consistency?